File: src/main/java/proto/common/Method.java
package proto.common;

public enum Method {

    // LOBBY
    CREATE_ROOM,
    ROOM_CREATED_SUCCESS,
    ROOM_CREATED_ERROR,
    GET_ROOMS,
    ROOMS_LIST,
    JOIN_ROOM_REQUEST,
    JOIN_ROOM_RESPONSE,
    LEAVE_ROOM,
    KICK_PLAYER,
    LOBBY_UPDATE,
    LOBBY_CHAT,

    // GAME
    START_GAME,
    GAME_STATE,
    PLAY_CARD,
    DRAW_CARD,
    SAY_UNO,
    GAME_CHAT,
    CHOOSE_COLOR,

    // SYSTEM
    PING,
    PONG,
    OK,
    ERROR
}

File: src/main/java/proto/common/Method.kt
package proto.common

enum class Method {
    // LOBBY
    CREATE_ROOM,
    ROOM_CREATED_SUCCESS,
    ROOM_CREATED_ERROR,
    JOIN_ROOM_REQUEST,
    JOIN_ROOM_RESPONSE,
    LEAVE_ROOM,
    KICK_PLAYER,
    LOBBY_UPDATE,

    // GAME
    START_GAME,
    GAME_STATE,
    PLAY_CARD,
    DRAW_CARD,
    SAY_UNO,
    CHOOSE_COLOR,
    FINISH_DRAWING,

    // SYSTEM
    PING,
    PONG,
    OK,
    ERROR
}